app/controllers/application_controller.rb

```diff
@@ -1,2 +1,41 @@
 class ApplicationController < ActionController::API
+  rescue_from(StandardError, with: :render_internal_error_response)
+  rescue_from(ArgumentError, with: :render_bad_request_response)
+  rescue_from(ActionController::ParameterMissing, with: :render_missing_parameter_response)
+  rescue_from(ActionController::BadRequest, with: :render_bad_request_response)
+  rescue_from(ActiveRecord::RecordInvalid, with: :render_record_invalid_response)
+  rescue_from(ActiveRecord::RecordNotFound, with: :render_not_found_response)
+
+  private
+
+  def render_internal_error_response(error)
+    render(json: { errors: [{ detail: 'Internal Server Error' }] }, status: :internal_server_error)
```

Comment:
To avoid exposing this message and the one below, I have changed how their returned to the front-end to ensure the message is instead, "Something went wrong"